README.md

```diff
@@ -702,11 +705,80 @@ Name        | Value example | Description
+`max_volume`                | 2                                                                     | Maximum volume level.
+`supports_auto_relock`      | true                                                                  | Defines if device supports auto-relock capability.
+`supports_hold_and_release` | false                                                                 | Defines if device supports hold and release capability.
+`supports_block-to-block`   | true                                                                  | Defines if device supports block-to-block capability.
+`supports_twist_assist`     | false                                                                 | Defines if device supports twist assist capability.
+
+#### Example
+
+```json
+{
+  "serv": "door_lock",
+  "type": "cmd.lock.set_configuration",
+  "val_t": "object",
+  "val": {
+    "operation_type": "timed",
+    "outside_handles_mode": {
+      "handle1": 1,
+      "handle2": 1,
+      "handle3": 1,
+      "handle4": 1,
+    },
+    "inside_handles_mode": {
+      "handle1": 1,
+      "handle2": 1,
+      "handle3": 1,
+      "handle4": 1,
+    },
+    "lock_timeout_minutes": 2,
+    "lock_timeout_seconds": 1,
+    "auto_relock_time": 60,
+    "hold_and_release_time": 55,
+    "block_to_block": true,
+    "twist_assist": false
+  },
+  "ver": "1",
+  "props": null,
+  "tags": null
+}
+
+```
+
+#### Notes
+
+Messages cmd.lock.set_configuration and evt.lock.configuration_report will look exactly the same according to value body.
+
+Explanation of cmd.lock.set_configuration interface value properties.
+
+- operation_type - One of sup_operation_types property within service. At the moment supported values are: 'timed' and 'constant'.
+
+- outside_handles_mode - Defines whether any of four outside handles can be opened locally, 1 means cannot open locally, 0 means can open locally. This field might be ommited, then all door handles can be opened locally.
+
+- inside_handles_mode - Defines whether any of four outside handles can be opened locally, 1 means cannot open locally, 0 means can open locally. This field might be ommited, then all door handles can be opened locally.
+
+- lock_timeout_minutes - This field is used to specify the time that a device must wait before returning to the secured mode when receiving timed operation modes. Values in range 0...253 must indicate the actual number of minutes. This field must be in range 0...253 if the operation_type field is set to 'timed'. This field might be ommited when operation_type field is set to 'constant'.
+
+- lock_timeout_seconds - This field is used to specify the time that a device must wait before returning to the secured mode when receiving timed operation modes. Values in range 0...59 must indicate the actual number of seconds. This field must be in range 0...59 if the operation_type field is set to 'timed'. This field might be ommited when operation_type field is set to 'constant'.
```

Comment:
min and max should preferably be mentioned somewhere in props

Reply:
Fixed, added it to props